app/src/main/java/xyz/whereuat/whereuat/utils/HttpRequestHandler.java

```diff
@@ -14,16 +14,30 @@
 import xyz.whereuat.whereuat.Constants;
 
 /**
- * Created by julius on 3/25/16.
+ * <p>Utilities for HTTP requests with the server. Uses the
+ * <a href="https://android.googlesource.com/platform/frameworks/volley">Volley</a> framework</p>
  */
 public class HttpRequestHandler {
     private static RequestQueue mRequestQ;
 
+    /**
+     * Constructor for the HttpRequestHandler. Instantiates a request queue for Volley
+     *
+     * @param context Context for the Volley request queue
+     */
     public HttpRequestHandler(Context context) {
         if (mRequestQ == null)
             mRequestQ = Volley.newRequestQueue(context);
     }
 
+    /**
+     * Method to build and initiate the POST request for requesting a new account from the server
+     *
+     * @param phone_number Phone number of the account
+     * @param success_listener Listener for a success (200) response from the server
+     * @param error_listener Listener for an error response from the server
+     * @return true if POST request is built successfully, false otherwise
```

Comment:
I've always been told to just give the `true` case for a boolean return because `false` is obviously just not the true case